### PR TITLE
Update cloud-native-security.md

### LINKED ISSUE #48264: Missing word in cloud native security page
**This is a Bug Report**

**Problem:**
I wrote [Cloud Native Security and Kubernetes](https://kubernetes.io/docs/concepts/security/cloud-native-security/) but I think I missed a word out.
In the paragraph:
> [Containers](https://kubernetes.io/docs/concepts/containers/) provide two things: isolation between different applications, and a mechanism to combine those isolated applications to run on the same host computer. Those two aspects, isolation and aggregation, mean that runtime security involves trade-offs and finding an appropriate balance.

the trade-offs parts seems to be repeated by “finding an appropriate balance“.

**Proposed Solution:**
Before “trade-offs“, add the word “identifying”

**Page to Update:**
https://kubernetes.io/docs/concepts/security/cloud-native-security/

**Additional Information:**
/good-first-issue
/triage accepted
/sig security
/language en

Anyone is welcome to send in a PR to fix this. If you've contributed to Kubernetes before, please consider leaving this one for a new contributor to pick up.

### PR DESCRIPTION
**Pull request by tbansod2018**
### Description
Added Missing word in cloud native security page

### Issue
Fixes #48264

Closes: #

### CODE REVIEW

File: content/en/docs/concepts/security/cloud-native-security.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ pay special attention to restricting misuse there.
 {{< glossary_tooltip text="Containers" term_id="container" >}} provide two
 things: isolation between different applications, and a mechanism to combine
 those isolated applications to run on the same host computer. Those two
-aspects, isolation and aggregation, mean that runtime security involves
+aspects, isolation and aggregation, mean that runtime security involves identifying
```
Comment (utkarsh-singh1) [inline content/en/docs/concepts/security/cloud-native-security.md:116]:
```suggestion
aspects, isolation and aggregation, mean that runtime security involves
identifying
```

File: content/en/docs/concepts/security/cloud-native-security.md [hunk 2]
```diff
@@ -113,7 +113,8 @@ pay special attention to restricting misuse there.
 {{< glossary_tooltip text="Containers" term_id="container" >}} provide two
 things: isolation between different applications, and a mechanism to combine
 those isolated applications to run on the same host computer. Those two
-aspects, isolation and aggregation, mean that runtime security involves identifying
+aspects, isolation and aggregation, mean that runtime security involves
+identifying trade-offs and finding an appropriate balance.
 trade-offs and finding an appropriate balance.
```
Comment (utkarsh-singh1) [inline content/en/docs/concepts/security/cloud-native-security.md:118]: @tbansod2018, remove this line as it repeats the previous one.

**Review by dipesh-rawat — COMMENTED** [review]
Thanks for the fix.
Congrats on your first PR 🎉

[Preview Page](https://deploy-preview-48266--kubernetes-io-main-staging.netlify.app/docs/concepts/security/cloud-native-security/)

/lgtm
/approve

**Pull request merged**